subcommand/lifecycle-sidecar/command_test.go

```diff
@@ -27,14 +42,6 @@ func TestRun_FlagValidation(t *testing.T) {
 			},
 			ExpErr: "-consul-binary must be set",
 		},
-		{
```

Comment:
does this not error anymore?

Reply:
Yeah, with Rebecca's suggestion to use time.Duration, the flags library handles it for us. Since we don't have any code for validation and parsing of this value anymore, we don't need this test.